### PR TITLE
t_client with AWS support

### PR DESCRIPTION
**Pull request by lstipakov**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,5 @@
 *.ovpn
 *.pid
 run.ps1
+*.log
+*.msi
```
Comment (flichtenheld) [inline .gitignore:5]: add missing newline at end of file

File: Start-AWSTest.ps1 [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+function Start-TestMachine() {
+    # Start AWS instance
+    Write-Host "Starting instance"
+
+    $instId = (New-EC2Instance `
+        -ImageId $IMAGE_ID `
+        -InstanceType $INSTANCE_TYPE `
+        -SubnetId $SUBNET_ID `
+        -SecurityGroupId $SECURITY_GROUPS).Instances[0].InstanceId
```
Comment (flichtenheld) [inline Start-AWSTest.ps1:50]: As discussed, please add basic set of tags here with -TagSpecification

File: .gitignore (hunk 1 above)
Comment (lstipakov) [reply .gitignore]: done!

**Review by flichtenheld — APPROVED** [review]
Looks good to me

**Pull request merged**